libcontainer/rootfs_init_linux.go

```diff
@@ -231,9 +231,9 @@ func doDockerDnsSwitch(oldDns, newDns string) error {
 
 // sysbox-runc:
 // Init performs container rootfs initialization actions from within the container's mount
-// namespace only. By virtue of only entering the mount namespace, Init has true
-// root-level access to the host and thus can perform operations that the container's init
-// process is not allowed to.
+// namespace only. By virtue of only entering a specific namespace (e.g. 'mount'), Init has
```

Comment:
Changes to this file are not required I think.

Reply:
This is to simply amend the comment which assumes that this logic is only relevant for the "mount" namespace, which is not correct as "switchDockerDns" case uses the network-ns. I just enhanced further in my latest patch.